docs/maps/lazarus.md

```diff
@@ -0,0 +1,40 @@
+<!-- replace _map_ with the actual map name -->
+<!-- change gamemode type for the Map data description  -->
+# Lazarus
+
+***
+
+#### ![lazarusicon](../assets/maps/lazarus/lazarus-icon.jpg)
+
+# Overview
+***
+- **Introduced:** v1.8.0
+- **Description:** A recreation of the Team Fortress 2 map
```

Comment:
Please change this description to describe the map. Not everyone knows what the TF2 map looks like.